curdleproofs/curdleproofs/whisk_interface.py

```diff
@@ -9,56 +9,72 @@
 from curdleproofs.curdleproofs_transcript import CurdleproofsTranscript
 from curdleproofs.opening import TrackerOpeningProof
 from curdleproofs.util import (
-    PointAffine as BLSG1Point,
+    PointAffine,
     affine_to_projective,
     Fr,
+    BufReader,
 )
-from py_ecc.optimized_bls12_381.optimized_curve import G1, normalize
+from py_ecc.optimized_bls12_381.optimized_curve import G1, normalize, multiply
+from py_ecc.bls.g2_primitives import (
+    G1_to_pubkey,
+    pubkey_to_G1,
+)
+from eth_typing import BLSPubkey
+from py_ecc.bls.hash import os2ip
 
+Bytes32 = NewType('Bytes32', bytes)
 
 class WhiskTracker:
-    r_G: BLSG1Point  # r * G
-    k_r_G: BLSG1Point  # k * r * G
+    r_G: BLSPubkey  # r * G
+    k_r_G: BLSPubkey  # k * r * G
 
-    def __init__(self, r_G: BLSG1Point, k_r_G: BLSG1Point):
+    def __init__(self, r_G: BLSPubkey, k_r_G: BLSPubkey):
         self.r_G = r_G
         self.k_r_G = k_r_G
 
 
 SerializedCurdleProofsProof = bytes
 
 
+def pubkey_to_G1c(pubkey: BLSPubkey) -> PointAffine:
```

Comment:
What does `pubkey_to_G1c` stand for? What is the `c` for?

Reply:
Not sure TBH, probably needed to differentiate the function somehow but the c suffix is a bit random. 
- Let's merge this one first to use a single pubkey_to_G1 function https://github.com/nalinbhardwaj/curdleproofs.pie/pull/13

Reply:
Removed pubkey_to_G1c from PR